testing/sdk_tasks.py

```diff
@@ -175,7 +175,27 @@ def get_completed_task_id(task_name):
 
     return tasks[0] if tasks else None
 
+def check_scheduler_relaunched(service_name: str, old_scheduler_task_id: str,
```

Comment:
Since the PR that changed the way that `check_task_relaunched` is implemented, this function has become out of date. Would it be possible to reuse `check_task_relaunched` in some way instead?